### DISCUSSION TITLE
I got warning when I open buffers or git status: invalid mapping for Y nil

### DISCUSSION THREAD

**Discussion by rofrol** [nvim-neo-tree/neo-tree.nvim#1324] [General]
I needed to disable it:

```lua
      buffers = {
        window = {
          mappings = {
            -- disable Y mapping
            ["Y"] = "noop"
          }
        },
      },
      git_status = {
        window = {
          mappings = {
            -- disable Y mapping
            ["Y"] = "noop"
          }
        },
      },
```

My config https://github.com/rofrol/LazyVim--starter

**Comment by cseickel** [comment]
It would help if you included your entire neo-tree config here. I looked at the linked config and did not see the config you posted here.

**Comment by rofrol** [comment]
Here is my config that disables `Y` mapping https://github.com/rofrol/LazyVim--starter/blob/301d8f3bd9a54901aaa6834f88e738e033fb1b11/lua/plugins/core.lua#L43

and here is plugin from LazyVim https://github.com/LazyVim/LazyVim/blob/a50f92f7550fb6e9f21c0852e6cb190e6fcd50f5/lua/lazyvim/plugins/editor.lua#L7

**Comment by cseickel** [comment]
I think the problem is that LazyVim defines that mapping at the global level, so you would have to put your override at the global level as well:

``` lua
      window = {
        mappings = {
          -- disable Y mapping
          ["Y"] = "noop"
        }
      },
```

**Comment by rofrol** [comment]
I think you have misunderstood me.

These mappings with noop are to disable warnings. And they work ok for this purpose. No need for global mappings.

**Reply by cseickel** [reply]
Oh, sorry, I thought you were asking why you were getting the error. Now I see you were just reporting how you solved it. I guess it's some kind of bug with the LazyVim starter.
